spec/cluster_holding_spec.rb

```diff
@@ -78,6 +78,20 @@ def new_submission(holding, date: Date.today)
         eq(h2.date_received)
     end
 
+    it "updates multiple existing holdings if cluster.large?" do
+      c.ocns << Services.large_clusters.ocns.first
+      c.save
+      described_class.new(h).cluster
+      described_class.new(h.clone).cluster
```

Comment:
This shouldn't add an additional holding, right?

Reply:
Maybe? `.cluster` is little more than `cluster.add_holdings` which is `.push_to_field`. 
`.update` performs the check for existing matching holdings and, with this PR, large clusters. 

Should `.cluster` check for one or both of those special cases? Should `.update` really just be renamed `.cluster`?   

bin/add_print_holdings.rb requires a -u flag for updates. Other than the initial load, I can't imagine a use case for `.cluster` as written.

Reply:
Let's get rid of `#cluster` (or just alias it to `#update`)